gulp-tasks/https-server.js

```diff
@@ -34,18 +34,19 @@ function handleRequest(req, res) {
   } else if(req.method === 'GET') {
     let url = req.url;
     console.log(`Url='${url}'`);
-    if (url.endsWith('/download-manager/rest/tc-accepted?downloadURL=/file/cdk-2.1.0.zip')) {
+    if (url.endsWith('/download-manager/rest/tc-accepted?downloadURL=/file/cdk-2.4.0.zip')) {
       res.end('true');
     } else if(url.endsWith('/favicon.ico')) {
       // not required
     } else {
       console.log('Request to download manager ');
       for (let prop in requirements) {
         let requirement = requirements[prop];
+        console.log(requirement.dmUrl);
         if(requirement.dmUrl && requirement.dmUrl.endsWith(url)
           || requirement.url && requirement.url.endsWith(url) ) {
           console.log('Issuing redirect ' + 'https://' + req.headers['host'] + '/' + prop);
```

Comment:
You might want to update the filename in the logging as well, for consistency's sake.